src/tests/ftest/launch.py

```diff
@@ -1,13 +1,10 @@
-#!/usr/bin/python3 -u
+#!/usr/bin/env python3
```

Comment:
I've stumbled upon a different issue where this aspect of the change is important so rather than close this PR I've attempted to make the change to pipeline-lib so we can move forward.